projects/plugins/jetpack/class.jetpack-plan.php

```diff
@@ -47,6 +47,7 @@ class Jetpack_Plan {
 				'whatsapp-button',
 				'social-previews',
 				'videopress',
+				'post-publish-qr-post-panel',
```

Comment:
I don't think we need this if the panel will be available to all sites on wpcom simple; we can bypass all kinds of plan checks if we know the feature will be free for everyone.

Reply:
When removing the plugin is not loaded :-/

Reply:
This looks tidy to me. Let's go like this and iterate later if there's a better alternative

Reply:
> Great news! One last step: head over to your WordPress.com diff, D76203-code, and commit it. Once you've done so, come back to this PR and add a comment with your changeset ID.

r241291-wpcom

Reply:
I opened #23290 to simplify things. As a result you shouldn't have to declare support on WordPress.com simple for the new feature.